src/runtime/pybuffer.cs

```diff
@@ -78,19 +85,28 @@ unsafe internal PyBuffer(PyObject exporter, int flags)
         public static long SizeFromFormat(string format)
         {
             if (Runtime.pyversionnumber < 39)
-                throw new NotSupportedException("GetPointer requires at least Python 3.9");
+                throw new NotSupportedException("SizeFromFormat requires at least Python 3.9");
             return (long)Runtime.PyBuffer_SizeFromFormat(format);
         }
 
         /// <summary>
         /// Returns true if the memory defined by the view is C-style (order is 'C') or Fortran-style (order is 'F') contiguous or either one (order is 'A'). Returns false otherwise.
         /// </summary>
         /// <param name="order">C-style (order is 'C') or Fortran-style (order is 'F') contiguous or either one (order is 'A')</param>
-        public bool IsContiguous(char order)
+        public bool IsContiguous(BufferOrderStyle order)
         {
             if (disposedValue)
                 throw new ObjectDisposedException(nameof(PyBuffer));
-            return Convert.ToBoolean(Runtime.PyBuffer_IsContiguous(_handle, order));
+
+            char style = 'C';
+            if (order == BufferOrderStyle.C)
+                style = 'C';
+            else if (order == BufferOrderStyle.Fortran)
+                style = 'F';
+            else if (order == BufferOrderStyle.EitherOne)
+                style = 'A';
+
+            return Convert.ToBoolean(Runtime.PyBuffer_IsContiguous(_gchandle.AddrOfPinnedObject(), style));
```

Comment:
Again, why can't you use `&_view` or `ref _view` everywhere you call `_gchandle.AddrOfPinnedObject()`? You would not need the `_gchandle` in the first place.

Reply:
Dont we need `gchandle` so it doesnt get garbage collected? And to garbage collect it in the `Dispose()` function by calling `gchandle.Free()`?

Reply:
I mean, wouldnt the garbage collertor collect the `byte[] rawData` because it thinks its unused without pinning it through a `GCHandle`?

Reply:
I am unsure why do you even need `rawData`. Why can't you just pass `&_view` or `ref _view`?

Reply:
@lostmsu Ok i tested it and `&_view` didnt work, but `ref _view` worked. I always thought `ref` and `&` work only for primitive and unmanaged types and not on managed types.